boards/native/native_sim/doc/index.rst

```diff
@@ -500,10 +500,14 @@ The following peripherals are currently provided with this board:
     :ref:`its section <nsim_per_disp_sdl>`.
 
 **CAN controller**
-  It is possible to use a host CAN controller with the native SockerCAN Linux driver. It can be
+  It is possible to use a host CAN controller with the native SocketCAN Linux driver. It can be
   enabled with :kconfig:option:`CONFIG_CAN_NATIVE_LINUX` and configured with the device tree binding
   :dtcompatible:`zephyr,native-linux-can`.
 
+  It is possible to specify CAN interface which will be used by the app using the ``--can-if``
```

Comment:
```suggestion
  It is possible to specify which CAN interface will be used by the app using the ``--can-if``
```

Reply:
Will fix after @henrikbrixandersen reviews the PR

Reply:
Done